features/shop/cart_with_shipping_fee.feature

```diff
@@ -0,0 +1,20 @@
+@ui-cart
+Feature: Cart shipping without taxes
+    In order to buy goods with correct shipping fees applied
+    As a Customer
+    I want to have correct shipping fees applied to my order
+
+    Background:
+        Given the store is operating on a single "France" channel
+        And default currency is "EUR"
+        And there is user "john@example.com" identified by "password123"
+        And store has a product "PHP T-Shirt" priced at "€100.00"
+        And store has "DHL" shipping method with "€10.00" fee
+        And I am logged in as "john@example.com"
+
+    Scenario: Adding proper shipping fee
+        Given I am logged in as "john@example.com"
+        And I add product "PHP T-Shirt" to the cart
```

Comment:
Yes, all Givens should be in past form I think. Or even:

```
I have product "PHP T-Shirt" in the cart
```

Because ``add`` / ``added`` is still an action. WDYT?